standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -3890,9 +3890,20 @@ public CompactionResponse compact2(String dbname, String tableName, String parti
     }
     cr.setType(type);
     cr.setProperties(tblproperties);
+    cr.setInitiatorId(hostname() + "-manual");
```

Comment:
why 'manual', rather than caller thread id?

Reply:
I want to distinguish between automatically and manually initiated compactions. This will be needed, when we want to add metric of the number of running Initiator instances and can be useful for debugging issues.

Reply:
gotcha!